terraform/modules/self-service/ecs.tf

```diff
@@ -11,6 +11,7 @@ locals  {
     cognito_user_pool_id  = "${aws_cognito_user_pool.user_pool.id}"
     asset_host            = "${var.asset_host}"
     asset_prefix          = "${element(split(":", var.image_digest),1)}/assets/"
+    sentry_dsn            = "arn:aws:ssm:eu-west-2:${data.aws_caller_identity.account.account_id}:parameter/${var.deployment}/ecs-app-shared/sentry-dsn"
```

Comment:
self-service should have its own DSN, otherwise this will go to the hub project but we should create our own sentry project.

Reply:
Ah ok, I thought we were sharing. Will look at setting one up

Reply:
sorry, I should have been clearer, we're sharing the sentry instance, but projects are different, hub, frontend and we need to add self-service...

Reply:
Okey dokey, have created a self service sentry DSN and stuck it in AWS SSM, FTW